src/app/modules/activities-management/store/activity-management.reducers.ts

```diff
@@ -56,6 +56,21 @@ export function activityManagementReducer(state: ActivityState = initialState, a
         message: 'Something went wrong creating activities!',
       };
     }
+    case ActivityManagementActionTypes.DELETE_ACTIVITY: {
+      return {
+        ...state,
+        message: 'Activity removed successfully!',
```

Comment:
you need to set the loading variable to true.

Reply:
done

Reply:
@daros10  you don't need to add "message: 'Activity removed successfully!'," that is only in DELETE_ACTIVITY_SUCCESS